lib/github/github_service.rb

```diff
@@ -11,12 +11,23 @@ def create_issue(feedback)
         feedback = Feedback.new(feedback)
 
         begin
-          client.create_issue(
+          # Create issue with email in body so that the email
+          # notification contains the email
+          issue = client.create_issue(
             GITHUB_REPO,
             issue_title(feedback),
             issue_body(feedback),
             assignee: 'va-bot', labels: 'User Feedback'
           )
+
+          # then update issue to remove email from GH issue
+          # for privacy concerns
+          client.update_issue(
```

Comment:
Mildly hacky 😉  but creative way around keeping the github notification flow without storing a reference id elsewhere or using a mailer with additional info that links to the github issue. Only concern would be if github actually does store this info and decides to make it retrievable (via API or UI) later on. Obviously still a private repo so maybe same emails in logs ATO rules apply?

Reply:
I'm not sure the logs ATO rules apply to github...pretty sure not, but @omgitsbillryan can you run this solution by #ato-support to be sure? I fear bad news, but better to know now than later.

Reply:
> Mildly hacky

You are too polite :stuck_out_tongue_winking_eye: 

> Only concern would be if github actually does store this info and decides to make it retrievable (via API or UI) later on

I agree.  This is valid point, to which I would say - we'll cross that bridge _if_ & _when_ we ever get to it.  I like the simplicity this gives us.  Less is more, avoid premature optimization, etc. etc.

Reply:
Hm...but we (or whoever is maintaining this at the time) will have no way of knowing that github changed something and we're now storing PII. This could be painful, but is there some kind of integration test we can add that exercises this code and then verifies that the email address is not present in the issue?

Reply:
I already know that the email is not present in the issue via manual testing.  Even after a hypothetical release of a new API to view comment revision history, the email would not be in the issue.  

I think you're poking at setting up a test for if & when Github allows one to view revision history.  I don't think this is possible since we don't know what that API would look like.

Reply:
Yeah, true...likely it would not show up in the feed as it exists today. 

OK, I'd say just follow up with Aaron and Brian on this request to email them the risks and mitigations and then forge ahead: https://dsva.slack.com/archives/C0PPJ1BHP/p1505946769000075?thread_ts=1505920963.000316&cid=C0PPJ1BHP